charts/opentelemetry-collector/examples/daemonset-and-deployment/rendered/service.yaml

```diff
@@ -42,4 +42,4 @@ spec:
   selector:
     app.kubernetes.io/name: opentelemetry-collector
     app.kubernetes.io/instance: example
-    component: standalone-collector
+    component: deployment-collector
```

Comment:
I would not change the labels at this point since it breaks the upgrade and forces user to reinstall the chart. deployment-collector doesn't sound much better then standalone-collector. I would recommend handling this separately

Reply:
Ok I can set it back

Reply:
@dmitryax done.